### PR TITLE
[lldb] Add a simple Swift/C++ interop step test

### PR DESCRIPTION
**Pull request by augusto2112**

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by augusto2112** [comment]
The first commit is a cherry-pick from llvm.org, so you don't need to review that, just the new test.

File: lldb/test/API/lang/swift/cxx_interop/forward/step-into-cxx/Makefile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+SWIFT_SOURCES := main.swift
+SWIFT_OBJC_INTEROP := 1
```
Comment (adrian-prantl) [inline lldb/test/API/lang/swift/cxx_interop/forward/step-into-cxx/Makefile]: Why do we need both?

**Review by adrian-prantl — APPROVED** [review]

File: lldb/test/API/lang/swift/cxx_interop/forward/step-into-cxx/Makefile (hunk 1 above)
Comment (augusto2112) [reply lldb/test/API/lang/swift/cxx_interop/forward/step-into-cxx/Makefile]: We don't 😅 removed

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by augusto2112** [comment]
@swift-ci test

**Pull request merged**